frontend/src/pages/NewRunV2.tsx

```diff
@@ -204,6 +205,19 @@ function NewRunV2(props: NewRunV2Props) {
       ? !cloneOrigin.recurringRun.no_catchup
       : true;
   const [needCatchup, setNeedCatchup] = useState(initialCatchup);
+  const [useLatestVersion, setUseLatestVersion] = useState(false);
+
+  useEffect(() => {
+    if (useLatestVersion) {
+      // Clear pipeline version when using latest
+      setPipelineVersionName('');
+      setRunName(
+        existingPipeline
+          ? 'Run of ' + existingPipeline.display_name + ' ' + new Date().toISOString()
```

Comment:
Is there a particular reason to use a different naming scheme when using the latest version? I think we could probably keep the existing naming pattern.